report/renderer/performance-category-renderer.js

```diff
@@ -227,45 +210,42 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       filmstripEl && timelineEl.append(filmstripEl);
     }
 
-    // Opportunities
-    const opportunityAudits = category.auditRefs
-        .filter(audit => this._classifyPerformanceAudit(audit) === 'load-opportunity')
-        .filter(audit => !ReportUtils.showAsPassed(audit.result))
-        .sort((auditA, auditB) => this._getWastedMs(auditB) - this._getWastedMs(auditA));
-
     const filterableMetrics = metricAudits.filter(a => !!a.relevantAudits);
```

Comment:
what about the metric filter?  it seems like we have all the info we need to resort when we filter to just FCP, etc.

Reply:
Good point. That might be best left as a follow up and the old metric filter is adequate for now. After this PR but before the release?

Reply:
sure. clearly something about the score/icons will also have to land post this-PR , pre-release. 
sg